src/document/DocumentAnnotator.ts

```diff
@@ -12,10 +13,31 @@ export default class DocumentAnnotator extends BaseAnnotator {
 
     managers: Map<number, Set<BaseManager>> = new Map();
 
+    selectionChangeTimer: NodeJS.Timer | null = null;
+
     constructor(options: Options) {
         super(options);
 
         this.addListener(Event.ANNOTATIONS_MODE_CHANGE, this.handleChangeMode);
+        document.addEventListener('selectionchange', this.handleSelectionChange);
```

Comment:
From what I've seen, the selectionchange event is really noisy, what do you think about debouncing the `handleSelectChange` event handler?